wp-content/plugins/core/src/Blocks/Types/Hero/Model.php

```diff
@@ -32,23 +25,36 @@ private function get_layout() {
 	/**
 	 * @return int|bool
 	 */
-	private function get_media() {
+	private function get_media_id() {
 		return $this->get( Hero::IMAGE, false );
 	}
 
 	/**
 	 * @return array
 	 */
-	private function get_content(): array {
+	private function get_cta_args(): array {
+		$cta = wp_parse_args( $this->get( Hero::CTA, [] ), [
+			'title'  => '',
+			'url'    => '',
+			'target' => '',
+		] );
```

Comment:
Dumb q, what is the purpose of this?

Reply:
if the CTA is for some reason the default (an empty array) it will add some default array keys. Saves me from doing a bunch of `isset` calls or `??`

Reply:
Ahh, cool, good idea